paper.md

```diff
@@ -50,35 +50,61 @@ event: BH2021
+
+## Creating a DPIA Knowledge Model
+
+The DPIA knowledge model is mainly inspired by the several existing documents, namely the [CNIL DPIA document template](https://www.cnil.fr/sites/default/files/atoms/files/cnil-pia-2-en-templates.pdf), the DPIA document template used by the Luxembourg Centre for Systems Biomedicine and [ICO guidance](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/data-protection-impact-assessments-dpias/how-do-we-do-a-dpia/#how)) on data protection.
 
-## Forming DPIA Knowledge Model
+The first chapter collects administrative information and description of method followed for during project's DPIA. Following chapters contain questions related to:
+
+* The identification of the need for a DPIA
+* The nature, scope, context and purposes of the processing of data, as well as the list of related responsibilities
+* The data processes and supporting assets
+* The proportionality and necessity of the processing
+* The measures taken to protect the rights of the subjects
+* The identification, analysis, and evaluation of the data risks factors and the measures and controls to mitigate those risks
+
+Although a risk assessment is very important part of a DPIA, our knowledge model is addressing it very briefly only by questions on general controls which are or will be in place. As risks generally depend on the used infrastructure and involved institutional units, they are globally applicable to all the processing under same institute. We propose process, where the risk assessment matrix can be generated by in-house or external professionals using dedicated tools (e.g. [Monarc](https://www.monarc.lu/)) and reused for each DPIA as a resource. For this purpose, our knowledge model includes question to provide link to such a document.
+
+Finally, our knowledge model was designed with user experience in mind. Complex branching logic, question description and guidance allow users to fill the questionnaire without additional help.
 
-*TODO: (Pinar), Vilém, Nene, and Paulette - what was the process, how you worked, what is the result*
 
 ## DPIA Document Template
 
 *TODO: Nene and Paulette - what was the process, how you worked, what is the result*
 
+In order to have the possibility to export the DPIA Knowledge model in a  document format which follow the structure of the CNIL Template, we have set up the DPIA Document Template.  It allows the different responses of the DPIA knowledge model to be collected in one document that can be exported in differents formats, e.g., HTML, MS WORD Document, etc... We used the [Template Development Kit (TDK)](https://github.com/ds-wizard/dsw-tdk) provided by DSW for the creation and the customization of the template; recommendations and hints can be found [here](https://docs.ds-wizard.org/en/latest/dev/templates.html).
```

Comment:
the second sentence "It allows the different responses... " can be imho deleted. It is describing purely what a DSW can do. I suggest to include more template specific information - does our KM contain all the information required by CNIL? Are there any answers not used by the template? Are there any answers which are aggregated? How is the administrative infromation mapped to the template.... etc..something on this topic - doenst have to answers all :)

Reply:
I have added small information on the template properties. 
@froggypaule Could you please take a quick look at this section and my proposed content above - if there is anything coming in your mind it would be great. I was not involved in the template implementation so my knowledge here is very limited. Also, if you dont have anything to add, its totally fine. I would say we can give it a go :)

Reply:
Hello @vildead here we go. I did a read through. (I did a review process, forgot how to properly review git commits!!)
Thanks for the opportunity, cheers!